src/nn/models.py

```diff
@@ -180,9 +180,7 @@ def __init__(self, im_channels=3):
         self.up4 = UpBlock(128)
         # output is now at 64x388x388
         self.conv_out = nn.Sequential(
-            nn.Conv2d(64, 2, kernel_size=1, padding=0),
-            nn.ReLU(),
```

Comment:
Also I'm not sure about how this works - I would assume you don't need relu+norm on the output layer